src/backend/joanie/signature/backends/lex_persona/client.py

```diff
@@ -0,0 +1,254 @@
+        headers = {"Authorization": f"Bearer {self.token}"}
+        params = {
+            "createDocuments": "true",
+            "ignoreAttachments": "false",
+            "signatureProfileId": str(self.signature_page),
+            "unzip": "false",
+            "pdf2pdfa": "auto",
+        }
+        # Get the pdf doc / later (#358 PR) we will use contract.document property
+        pdf_file_path = os.path.join(
+            "/app/joanie/signature/backends/lex_persona", "template.pdf"
+        )
+        with open(pdf_file_path, "rb") as pdf_file:
+            file_data = {
+                "document": (
+                    os.path.basename(pdf_file_path),
+                    pdf_file,
+                    "application/pdf",
+                )
+            }
+            response = requests.post(
+                url, params=params, headers=headers, files=file_data, timeout=3
+            )
+        return response.json()
+
+    def _update_workflowstatus_to_start(self):
+        """
+        Start the workflow process once it has been created
+        and the file has been uploaded to the workflow.
+        """
+        url = f"{self.base_url}/api/workflows/{self.workflow_id}"
+        payload = {"workflowStatus": "started"}
+        response = requests.patch(url, json=payload, headers=self.headers, timeout=3)
+
+        return response.json()
+
+    def _create_invitation_link_to_sign(self, recipient_email: str):
+        """
+        Method to create the invitation link which leads the signer
+        to the consent page where the signer can see all the documents
+        which require a signature.
+
+        Note :
+            - recipient_email must be a valid email that has been
+            used to create a workflow as the receivers.
+
+            - workflow_id must be the workflow that the user is
+            concerned.
+        """
+        url = f"{self.base_url}/api/workflows/{self.workflow_id}/invite"
+        payload = {"recipientEmail": recipient_email}
+        response = requests.post(url, headers=self.headers, json=payload, timeout=3)
+        return response.json()
+
+    def _retrieve_all_files_to_sign(self, recipient_email: str):
+        """
+        For organization's owners : in order to get all
+        the workflows that are ongoing and the ones that
+        are completed.
+
+        The advantage of this endpoint : it will regroup
+        every workflows that is linked to the recipient
+        email address.
+
+        The API filters them out according to the given
+        email value.
+        """
+        url = f"{self.base_url}/api/workflows/"
+        params = {
+            "item.currentRecipientEmails": recipient_email,
+        }
+        response = requests.get(url, headers=self.headers, params=params, timeout=3)
+        return response.json()
+
+    def _sign_specific_selection_of_workflow(self, workflow_ids: list):
+        """
+        WIP
```

Comment:
What's WIP here?

Reply:
I am currently waiting for an answer from the signature provider about this endpoint.
It seems to be working **only** when the session user is taking part into the signature. I shall investigate further.

Reply:
Got some feedback from the signature provider about this endpoint. We will fix it tomorrow.
Small information : it was indeed a wrong `consent_page_Id` that was misconfigured on their side, which explains why the **url** in the response from the API was **incorrect** and led to their **dev instance** instead of the **testing instance**.

Meanwhile, they will generate a new `consentPageId` with the correct access rights to perform this action tomorrow. More to come next ...